schema.ts

```diff
@@ -568,6 +568,31 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * Enabling options that can be used with a {@link ImageSchema}.
+ */
+ export enum ImageState {
+  // Image styling is disabled.
+  Disabled = 'disabled',
+  // Image styling is applied.
+  Default = 'default',
+ }
+
+/**
+ * A schema representing a return value or object property that is provided as a string,
+ * which Coda should interpret as an image.
+ */
+ export interface ImageSchema extends BaseStringSchema<
+  ValueHintType.ImageReference | ValueHintType.ImageAttachment> {
+  /** Instructs Coda to render this value as an Image. */
+  codaType: ValueHintType.ImageReference | ValueHintType.ImageAttachment;
+
+  // Boolean specifying whether or not to add outline to rendered images. Defaults to true.
```

Comment:
It's not a boolean anymore.
Also by single-line I meant surround it with `/** text */` so that it is still a jsdoc comment, not a simple comment.

Reply:
ahh I see I was trying to figure out the proper comment styling